src/main/java/org/opentripplanner/apis/transmodel/model/stop/QuayType.java

```diff
@@ -360,18 +350,8 @@ public static GraphQLObjectType create(
         GraphQLFieldDefinition
           .newFieldDefinition()
           .name("stopType")
-          .type(Scalars.GraphQLString)
-          .dataFetcher(environment -> {
-            StopLocation stopLocation = environment.getSource();
-            if (stopLocation instanceof RegularStop) {
-              return "regular";
-            } else if (stopLocation instanceof AreaStop) {
-              return "flexible_area";
-            } else if (stopLocation instanceof GroupStop) {
-              return "flexible_group";
-            }
-            return null;
-          })
+          .type(StopTypeMapper.STOP_TYPE)
+          .dataFetcher(env -> ((StopLocation) env.getSource()).getStopType())
```

Comment:
Changing the type here might not be backwards compatible. I like the Type mapper, and still think we can use it here, but not sure if we can change from string to enum type.

Reply:
Made it a String type again.